.pre-commit-config.yaml

```diff
@@ -6,15 +6,17 @@ repos:
     rev: v0.1.2
     hooks:
       - id: ruff
-        args: ["--config", "pyproject.toml"]
+        args:
+          - --config=pyproject.toml
         exclude: "{{cookiecutter.project_slug}}/src/{{cookiecutter.package_name}}"
       - id: ruff-format
-        args: ["--config", "pyproject.toml"]
+        args:
+          - --config=pyproject.toml
   - repo: https://github.com/codespell-project/codespell
     rev: v2.2.6
     hooks:
       - id: codespell
-        args: [--write-changes]
```

Comment:
@samcunliffe they key is to just not have this line. I've used it in `UCL-MIRSG` repos recently to great effect.